ppb/__init__.py

```diff
@@ -1,22 +1,41 @@
 import logging
+import warnings
 from typing import Callable
 
 from ppb import events
 from ppb_vector import Vector
 from ppb.engine import GameEngine
 from ppb.scenes import BaseScene
-from ppb.sprites import BaseSprite
+from ppb.sprites import Sprite
 from ppb.systems import Image
 from ppb.systems import Sound
 
 __all__ = (
     # Shortcuts
-    'Vector', 'BaseScene', 'BaseSprite', 'Image', 'Sound', 'events',
+    'Vector', 'BaseScene', 'BaseSprite', 'Image', 'Sprite', 'Sound',
+    'events',
     # Local stuff
     'run', 'make_engine',
 )
 
 
+class BaseSprite(Sprite):
+    """
+    A stub that raises a deprecation warning when a user uses
+    ``ppb.BaseSprite.``
+    """
+    __warning = """Using ppb.BaseSprite is deprecated.
+
+    You probably want ppb.Sprite. If you're wanting to use BaseSprite and
+    mixins to change what features your sprites have, import
+    ppb.sprites.BaseSprite.
+    """
+
+    def __init__(self, **kwargs):
```

Comment:
Let's move this warning to `__init_subclass__()`

Reply:
I'd worry about that since you can instantiate BaseSprite directly to build with.

Reply:
Oh fair.